projects/demo-integrations/cypress/tests/angular/predicate.cy.ts

```diff
@@ -13,4 +13,16 @@ describe('@maskito/angular | Predicate', () => {
 
         cy.get('@name').focus().type('12341234abcd12341234').should('have.value', 'ABCD');
```

Comment:
This old test fails.

See [job's logs](https://github.com/Tinkoff/maskito/actions/runs/5062105791/jobs/9094853389?pr=316).

```
  @maskito/angular | Predicate
    1) can detect run-time changes
    ✓ supports asynchronous predicate (1160ms)


  1 passing (14s)
  1 failing

  1) @maskito/angular | Predicate
       can detect run-time changes:

      Timed out retrying after 10250ms
      + expected - actual

      -'1234 1234 1234 1234'
      +'ABCD'

```